nook/functions/common/python/claude_cli_client.py

```
@@ -0,0 +1,335 @@
+            
+        except subprocess.TimeoutExpired as e:
+            raise ClaudeCLITimeoutError(f"Claude CLI command timed out after {self.config.timeout}s") from e
+        except subprocess.CalledProcessError as e:
+            error_msg = e.stderr.strip() if e.stderr else "Unknown error"
+            raise ClaudeCLIProcessError(f"Claude CLI command failed: {error_msg}") from e
+    
+    def generate_content(
+        self, 
+        contents: str, 
+        system_instruction: Optional[str] = None,
+        **kwargs
```

Ruff (0.12.2)

Unused method argument: `kwargs`

(ARG002)